triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -790,6 +838,15 @@ object Runner extends StrictLogging {
   val maxParallelSubmissionsPerTrigger = 8
   val maxTriesWhenOverloaded = 6
 
+  /** Maximum number of in-flight commands that we shall allow *before* the ledger
+    *  client automatically fails submit requests
+    */
+  val maxInFlightCommands = 42
+
+  /** Used to control rate at which we throttle ledger client submission requests */
+  val maxSubmitRequests = 100
```

Comment:
Community edition back pressure at `100` I will go with 20 or max 50.
```suggestion
  val maxSubmitRequests = 50
```

Anyway this should be configurable. (other PR).